src/index.js

```diff
@@ -0,0 +1,121 @@
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+      UI.showAlert('Task Removed', 'success');
+      UI.delay();
+    }
+  });
+};
+window.editTask = (taskNumber) => {
+  let toBeEdited;
+  const tasks = Task.getTask();
+  tasks.forEach((task, index) => {
+    if (Number(task.now) === taskNumber) {
+      toBeEdited = task;
+      tasks.splice(index, 1);
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  });
+  document.getElementById(taskNumber);
+  document.getElementById('task-form').classList.remove('d-none');
+  document.getElementById('task-form').classList.add('d-block');
+  document.querySelector('#titletwo').value = toBeEdited.title;
+  document.getElementById('description').value = toBeEdited.description;
+  document.querySelector('#date').value = toBeEdited.date;
+};
+const span = document.getElementsByClassName('close')[0];
+span.onclick = function () {
+  document.getElementById('task-form').classList.remove('d-block');
+  document.getElementById('task-form').classList.add('d-none');
+};
+document.getElementById('project-form').addEventListener('submit', (e) => {
+  // Prevent actual submit
+  e.preventDefault();
+  const title = document.querySelector('#title').value;
+  if (title === '') {
+    UI.showAlert('Please fill in all fields', 'danger');
+    UI.delay();
+  } else {
+    const project = new Project(title, UI.countProject());
+    UI.storeProjectName(project);
+    UI.addProjectName();
+    UI.showAlert('project added successfuly', 'success');
+    setTimeout(() => {
+      window.location.reload();
+    }, 3000);
+  }
+});
+document.getElementById('task-form').addEventListener('submit', (e) => {
+  // Prevent actual submit
+  e.preventDefault();
+  const title = document.querySelector('#titletwo').value;
+  const description = document.getElementById('description').value;
+  const date = document.querySelector('#date').value;
+  const priority = document.querySelector('input[name="priority"]:checked').value;
+  if (title === '' || description === '' || date === '' || priority === undefined) {
+    UI.showAlert('Please fill in all fields', 'danger');
+    setTimeout(() => {
+      window.location.reload();
+    }, 3000);
+  } else {
+    const giveMeActiveProject = UI.getSelected();
+    const giveStoredProjects = UI.getProjectName();
+    if ((giveMeActiveProject.length === 0) || (giveMeActiveProject[0].length === 0)) {
+      const task = new Task(title,
+        description,
+        date,
+        priority,
+        Date.now().toString(),
+        giveStoredProjects[0].name);
+      Task.storeTask(task);
+    } else {
+      const task = new Task(title,
+        description,
+        date,
+        priority,
+        Date.now().toString(),
+        giveStoredProjects[Number(giveMeActiveProject[0])].name);
+      Task.storeTask(task);
+    }
+    UI.showAlert('Task added successfuly', 'success');
+    UI.delay();
+  }
+});
+//  UI.renderDefault();
```

Comment:
## Avoid committing and pushing commented out code.
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use `git history` instead. More on this on this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/).